scene/2d/remote_transform_2d.cpp

```diff
@@ -31,7 +31,7 @@
 #include "remote_transform_2d.h"
 #include "scene/scene_string_names.h"
 
-void RemoteTransform2D::_update_cache() {
+void RemoteTransform2D::force_update_cache() {
```

Comment:
I would keep the internal method private as `_update_cache`, but add a public `force_update_cache` for the binding that calls `_update_cache`.